### PR TITLE
fix LED and "Disabled" Problem in Device Dialog

### PR DESCRIPTION
**Pull request by AlphaLima**

### CODE REVIEW

**Comment by AlphaLima** [comment]
Important fix for device setup

**Comment by brunotl** [comment]
I don't what is fixed because that work better without fix ...

**Comment by AlphaLima** [comment]
-MAXNUMDEVICES removed because we already have the same definition NUMDEV.
-When more than two devices are enabled (A, B and F) and we restart LK Device B was disabled.
-The LED indicators didn't show the enabled devices.

File: Common/Source/Dialogs/dlgConfiguration.cpp [hunk 1]
```diff
@@ -4751,18 +4750,27 @@ void InitDlgDevice(WndForm *pWndForm) {
...
-  for(unsigned i = 0; i < MAXNUMDEVICES; ++i) {
+  for(unsigned i = 0; i < NUMDEV; i++) {
     TCHAR szWndName[5];
     _stprintf(szWndName, _T("cmd%c"), _T('A')+i);
     WindowControl * pWnd = pWndForm->FindByName(szWndName);
     if(pWnd) {
       pWnd->SetWidth(w);
       pWnd->SetLeft(lx);
+
+      if(_tcslen(dwDeviceName[i]) == 0)
```
Comment (brunotl) [inline Common/Source/Dialogs/dlgConfiguration.cpp]: that's wrong, device name can be "DISABLED"

**Pull request closed without merging**